tox.ini

```diff
@@ -111,7 +111,7 @@ passenv = MOLECULE_* LSR_MOLECULE_* HOME
 changedir = {toxinidir}
 deps =
     {env:LSR_ANSIBLE_DEP:ansible}
-    docker
+    'docker<4.3'
```

Comment:
Why is docker needed? The other commit in this PR makes it sound like podman will be used instead.